docs/cookbook/getSortedLocalDateTimes.mjs

```diff
@@ -0,0 +1,24 @@
+import assert from "assert";
+/**
+ * getSortedLocalDateTimes will sort an array of zoneless Temporal.DateTime instances by the
+ * corresponding local date and time of day (e.g., for building a conference schedule).
+ *
+ *
+ * @param {Temporal.DateTime[]} dateTimes This is a Date instance
+ * @param {boolean} direction ascending or descending order
+ * @returns {Temporal.DateTime[]} Temporal.Absolute instance
+ */
+function getSortedLocalDateTimes(dateTimes, reverse = false) {
+    let newDateTimes = dateTimes.sort(Temporal.DateTime.compare);
```

Comment:
It's generally preferable to avoid side effects, in my opinion.
```suggestion
    let newDateTimes = Array.from(dateTimes).sort(Temporal.DateTime.compare);
```